modules/core/shared/src/main/scala/lucuma/core/model/EmissionLine.scala

```diff
@@ -28,18 +31,20 @@ final case class EmissionLine[T](
    *   `Integrated` or `Surface`
    */
   def to[T0](implicit conv: TagConverter[LineFlux[T], LineFlux[T0]]): EmissionLine[T0] =
-    EmissionLine[T0](lineWidth, lineFlux.toTag[LineFlux[T0]])
+    EmissionLine[T0](lineWidth, lineFlux.map(_.toTag[LineFlux[T0]]))
 }
 
 object EmissionLine {
-  implicit def eqEmissionLine[T]: Eq[EmissionLine[T]] =
+  type LineFluxOverTime[T] = NonEmptyMap[Instant, Measure[PosBigDecimal] Of LineFlux[T]]
```

Comment:
I wondered why we don't add a `LineFluxMeasure` to parallel the `BrightnessMeasure`.

Reply:
@rpiaggio ?